tests/e2e-pw/specs/shopper/shopper-subscriptions-purchase-no-signup-fee.spec.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import test, { expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { shouldRunSubscriptionsTests } from '../../utils/constants';
+import { describeif, getMerchant, getShopper } from '../../utils/helpers';
+import RestAPI from '../../utils/rest-api';
+import { config } from '../../config/default';
+import { goToSubscriptions, goToOrder } from '../../utils/merchant-navigation';
+import {
+	fillCardDetails,
+	placeOrder,
+	setupCheckout,
+} from '../../utils/shopper';
+import { goToProductPageBySlug } from '../../utils/shopper-navigation';
+
+const productName = 'Subscription no signup fee product';
+const productSlug = 'subscription-no-signup-fee-product';
+
+const customerBillingConfig = config.addresses.customer.billing;
+let orderId;
+
+describeif( shouldRunSubscriptionsTests )(
+	'Shopper Subscriptions Purchase No Signup Fee',
+	() => {
+		test.beforeAll( async ( {}, { project } ) => {
+			const restApi = new RestAPI( project.use.baseURL );
+			await restApi.deleteCustomerByEmailAddress(
+				customerBillingConfig.email
+			);
+		} );
+		test( 'It should be able to purchase a subscription without a signup fee', async ( {
```

Comment:
Do we actually need to test the whole process or is it okay to just use a short hand like `placeOrderWithOptions( shopperPage, { productId: 88 } )`?

Reply:
The "place order" button is different in subscriptions, which required me to do it like this IIRC.